### PR TITLE
feat: INT-2315 transformed the gitignore files as part of the mocha/vitest/migrate-configuration codemod

### PR DESCRIPTION
**Pull request by grzpab**

### CODE REVIEW

**Review by r4zendev — APPROVED** [review]

File: codemods/mocha/vitest/migrate-configuration/src/index.ts [hunk 1]
```diff
@@ -149,6 +154,26 @@ export const repomod: Filemod<Record<string, never>, Record<string, never>> = {
 			};
 		}
 
+		if (path.endsWith('.gitignore')) {
+			const expressions = data.split('\n');
+
+			if (
+				expressions.some(
+					(expression) => expression.trim() === 'coverage',
+				)
```
Comment (r4zendev) [inline codemods/mocha/vitest/migrate-configuration/src/index.ts:163]: mb `includes('coverage')`?

Comment (grzpab) [reply codemods/mocha/vitest/migrate-configuration/src/index.ts:163]: I think we should target the `coverage` itself, not `coverage/a` for instance.

Comment (r4zendev) [reply codemods/mocha/vitest/migrate-configuration/src/index.ts:163]: i mean cases like `**/coverage`, `project-name/coverage`

Comment (grzpab) [reply codemods/mocha/vitest/migrate-configuration/src/index.ts:163]: so when it ends with coverage, it's correct?

Comment (r4zendev) [reply codemods/mocha/vitest/migrate-configuration/src/index.ts:163]: hm, yeah, probably endsWith should do

**Pull request merged**